lib/org/chromium/webidl/LocalGitRunner.js

```diff
@@ -119,6 +121,16 @@ foam.CLASS({
       value: true,
     },
     {
+      class: 'FObjectProperty',
+      of: 'foam.dao.DAO',
+      documentation: `If provided, all IDL outputs will be forwarded to given
+          DAO (for persistent storage).`,
+      name: 'outputDao',
+      factory: function() {
+        return this.dao;
```

Comment:
Ah, so initially I was worried that FOAM would return `undefined` if `outputDao` was not defined. This would trigger an error (something along the lines that factory returned `undefined`). It seems that if `outputDao` is not provided, the value is defaulted to `null`, so it seems like it is fine without `|| null`.

I will remove it from the other one. Thanks for catching it.